.all-contributorsrc

```diff
@@ -916,6 +916,19 @@
         "tutorial"
       ]
     },
+    {
+      "login": "VirginieLemaire",
+      "name": "Virginie",
+      "avatar_url": "https://avatars.githubusercontent.com/u/82816247?v=4",
+      "profile": "https://github.com/VirginieLemaire",
+      "contributions": [
+        "code",
+        "data",
+        "doc",
+        "ideas",
+        "translation"
+        ]
+    },
```

Comment:
- Move your profile details (line 919-931) to the end of the list, after "Ajiboso-Adeola".
- Delete the comma in line 931.
- Add a comma in the closing bracket's of "Ajiboso-Adeola" on line 945.

Reply:
ok